Cubical/Homotopy/EilenbergSteenrod.agda

```diff
@@ -0,0 +1,64 @@
+{-# OPTIONS --cubical --no-import-sorts --safe #-}
+
+module Cubical.Homotopy.EilenbergSteenrod where
+
+{-
+This module contains the Eilenberg-Steenrod axioms for ordinary reduced cohomology theories with binary additivity.
+The axioms are based on the ones given in Cavallo's MSc thesis (https://www.cs.cmu.edu/~ecavallo/works/thesis15.pdf) and Buchholtz/Favonia (2018)
+-}
+
+
+open import Cubical.Foundations.Prelude
+open import Cubical.Foundations.Pointed
+open import Cubical.Foundations.Function
+open import Cubical.Foundations.Equiv
+open import Cubical.HITs.Wedge
+open import Cubical.HITs.Pushout
+open import Cubical.HITs.Susp
+
+open import Cubical.Data.Nat
+open import Cubical.Data.Bool
+open import Cubical.Data.Sigma
+open import Cubical.Data.Int
+
+open import Cubical.Algebra.Group
+open import Cubical.Algebra.AbGroup
+open GroupEquiv
+open GroupHom
+
+private
+  variable
+    ℓ ℓ' ℓ'' : Level
+    A : Type ℓ
+    B : Type ℓ'
+
+cofib : (f : A → B) → Type _
+cofib f = Pushout (λ _ → tt) f
+
+cfcod : (f : A → B) → B → cofib f
+cfcod f = inr
+
+suspFun : (f : A → B) → Susp A → Susp B
```

Comment:
This should go into the Susp files. You could also use it to make https://github.com/agda/cubical/blob/master/Cubical/HITs/Susp/Properties.agda#L48 nicer